bleak/backends/bluezdbus/client.py

```diff
@@ -904,6 +916,27 @@ async def write_gatt_descriptor(
             "Write Descriptor %s | %s: %s", descriptor.handle, descriptor.obj[0], data
         )
 
+    def _read_notify_fd(self, fd, callback):
+        loop = asyncio.get_running_loop()
+        os.set_blocking(fd, False)
```

Comment:
We shouldn't actually need to make it non-blocking since this callback is only called when there is actually data ready.